pkg/operator/resources/cluster/controller.go

```diff
@@ -274,9 +274,7 @@ func getControllerClusterPolicyRules() []rbacv1.PolicyRule {
 				"openstackvolumepopulators",
 			},
 			Verbs: []string{
-				"get",
-				"list",
-				"watch",
+				"*",
```

Comment:
@bennyz hi. I remember having global perms was not recommended. Refer - https://issues.redhat.com/browse/CNV-22907 and https://issues.redhat.com/browse/CNV-32812

Reply:
I think I have to do this to because of how `pkg/operator/resources/cluster/rbac.go` is set

There is also:
```
		{
			APIGroups: []string{
				"cdi.kubevirt.io",
			},
			Resources: []string{
				"*",
			},
			Verbs: []string{
				"*",
			},
},
```
in L#169

But I'm open to suggestions

Reply:
Would you mind unifying those two with a comment that explains the motivation?
Some thing along the lines off, "*" permissions are bad, but logically it's clear that CDI can do anything on CDI resources